specification/containerregistry/resource-manager/Microsoft.ContainerRegistry/stable/2025-04-01/containerregistry.json

```diff
@@ -4374,8 +4347,8 @@
     "SyncProperties": {
       "description": "The sync properties of the connected registry with its parent.",
       "required": [
-        "messageTtl",
-        "tokenId"
+        "tokenId",
```

Comment:
Is it auto-generated? why do we need to change it?

Reply:
I am just matching it with the previously released 2025-03-preview swagger. It is not auto-generated. We manually add it